.gitignore

```diff
@@ -86,15 +87,15 @@ __pycache__/
 target/
 /test/
 /tmp/
-TAGS
-TAGS.emacs
-TAGS.vi
+tags*
+TAGS*
```

Comment:
maybe `TAGS` and `TAGS.*`? this "open-ended glob" makes me a touch nervous for some silly reason

Reply:
Fixed this.